src/reducers/user.reducer.js

```diff
@@ -41,6 +42,20 @@ export function fetchUser() {
     };
 }
 
+export const loginUser = credentials => dispatch => {
+    dispatch(requestUser());
+    return postJson('/auth/login', credentials)
+        .then(user => dispatch(receiveUser(user)))
+        .catch(() => dispatch(receiveUser(null)));
+};
+
+export const registerUser = ({ email, password }) => dispatch => {
```

Comment:
why do we use object destructuring and than create it back?
it is better to just pass it as it it (as you do that above)